src/main/scala/microsites/MicrositeKeys.scala

```diff
@@ -450,6 +449,9 @@ trait MicrositeAutoImportSettings extends MicrositeKeys {
     makeMultiversionMicrosite := {
       Def.sequential(createMicrositeVersions, clean, makeVersionedMicrosite, moveMicrositeVersions)
     }.value,
+    ghpagesPrivateMappings := {
+      sbt.Path.allSubpaths((target in makeSite).value).toList
```

Comment:
I guess this is what makes the trick for both `publishMicrosite` and `publishMultiversionMicrosite`, correct?

Reply:
Yes it is!

`publishMicrosite` and `publishMultiversionMicrosite` are just different compositions of other tasks, with both of them including `pushSite`. This task will be the one affected by this `ghpagesPrivateMappings`, where in the case of using `GHPagesPlugin`, instead of recreate the site, the content to be uploaded will be read from `target in makeSite` (i.e. _project/target/site_)

Probably in the future we might want to simplify the command, and just use one approach or the other depending on the settings established by the user (e.g. `micrositeVersionList` values)

I ended up defining the task directly, instead of using an auxiliary one, which in the end it was only adding boilerplate IMO.

Reply:
🚀